app/app.go

```diff
@@ -567,7 +557,7 @@ func NewWasmApp(
 		appCodec,
 		runtime.NewKVStoreService(keys[icahosttypes.StoreKey]),
 		app.GetSubspace(icahosttypes.SubModuleName),
-		app.IBCFeeKeeper, // use ics29 fee as ics4Wrapper in middleware stack
+		app.IBCKeeper.ChannelKeeper,
 		app.IBCKeeper.ChannelKeeper,
```

Comment:
This `app.IBCKeeper.ChannelKeeper, app.IBCKeeper.ChannelKeeper,` looks fishy

Reply:
It is correct here though, as there is no stack (porttypes.ICS4Wrapper), so we can just use the channelkeeper as the stack.